PrestaShop 1.6 release 3.9.0 (#2594)
MultiSafepay/docs: content/integrations/prestashop-1-6.md

1. Unpack the contents of the .zip archive and upload the **Modules** folder via SFTP to the PrestaShop root directory, merging the two folders.
2. Sign in to your PrestaShop 1.6 <<glossary:backend>>.
3. Go to **Modules and services** > **Payments and gateways**.  
   **Note:** You must install and configure the MultiSafepay Core Module (MultiSafepay) because all payment methods require certain settings and/or the API key in the core module.
4. In the next screen, proceed with the installation.
5. Enter your [API key](/docs/sites#site-id-api-key-and-security-code), and then click **Save**.
6. On the **Payments** tab, enable the relevant payment methods.
   <br>

## Upgrading to 3.9.x

> Note that the component and tokenization configurations have been moved to payment method level settings.\

Reviewer: ```suggestion
The component and tokenization configurations have been moved to the payment-method level settings.\
```

> Please make sure configuration is still correct by navigating to **Modules > Module Manager > Select your payment method**.

Reviewer: ```suggestion
To check the configuration is still correct, go to **Modules** > **Module manager** > **Select your payment method**.
```
